autoload/vcsjump.vim

```diff
@@ -10,7 +10,7 @@ endfunction
 
 function! vcsjump#jump(command) abort
   let l:command=join(map(split(a:command), 'shellescape(v:val)'))
-  cexpr system(s:jump_path . ' ' . l:command . ' 2> /dev/null')
+  cexpr system(s:jump_path . ' ' . l:command . ' ' . expand("%:p:h") . ' 2> /dev/null')
```

Comment:
A couple things:

- I'd want to `shellescape()` the parameter you've added here (consider what might happen if you're looking at a file that has a space anywhere in the path; the shell will split that up into multiple arguments unless you escape).
- Tacking the current directory on the end here will likely conflict with commands like `diff` (which can take additional arguments) and `grep` (which always take additional arguments).